tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -979,9 +991,9 @@ void test_spec_replacement_field() {
     test_pointer_specs<charT>();
     test_string_specs<charT>();
 }
-

```

Comment:
Style nitpick: The newline between these functions should be retained. (The general convention is to have a newline between all functions, except for short highly repetitive functions.) However, this isn't worth resetting testing.